### PR TITLE
PLFM-1969: ACLs for Submissions; PLFM-2015: READ_PRIVATE_SUBMISSION permission

### PR DESCRIPTION
**Pull request by bkng**

### CODE REVIEW

File: services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/SubmissionManagerImpl.java [hunk 1]
```diff
@@ -229,15 +218,11 @@ public void deleteSubmission(UserInfo userInfo, String submissionId) throws Data
 			throws DatastoreException, UnauthorizedException, NotFoundException {
 		EvaluationUtils.ensureNotNull(evalId, "Evaluation ID");
 		UserInfo.validateUserInfo(userInfo);
-		String principalId = userInfo.getIndividualGroup().getId();
-		
-		if (!authorizationManager.canAccess(
-				userInfo, evalId, ObjectType.EVALUATION, ACCESS_TYPE.UPDATE))
-			throw new UnauthorizedException("User Principal ID" + principalId + " is not authorized to adminster Evaluation " + evalId);
+		validateEvaluationAccess(userInfo, evalId, ACCESS_TYPE.READ_PRIVATE_SUBMISSION);
```
Comment (eric-wu) [inline services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/SubmissionManagerImpl.java:221]: Should be DELETE?

File: services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/SubmissionManagerImpl.java [hunk 2]
```diff
@@ -314,20 +299,17 @@ public URL getRedirectURLForFileHandle(UserInfo userInfo,
 		Submission submission = getSubmission(userInfo, submissionId);
 		
 		// validate permissions
-		boolean isEvaluationAdmin = authorizationManager.canAccess(userInfo,
-				submission.getEvaluationId(), ObjectType.EVALUATION, ACCESS_TYPE.UPDATE); // TODO: change to READ_PRIVATE_SUBMISSION
-		if (!isEvaluationAdmin) {
-			throw new UnauthorizedException("Insufficient priveliges to " +
-					"download files from this Evaluation");
+		String ownerId = submissionDAO.get(submissionId).getUserId();
+		boolean isSubmissionOwner = ownerId.equals(userInfo.getIndividualGroup().getId());
```
Comment (eric-wu) [inline services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/SubmissionManagerImpl.java]: should be restricted to eval. admins only

File: services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/SubmissionManagerImpl.java [hunk 3]
```diff
@@ -209,16 +204,10 @@ public SubmissionStatus updateSubmissionStatus(UserInfo userInfo, SubmissionStat
...
 		
 		Submission sub = submissionDAO.get(submissionId);		
 		String evalId = sub.getEvaluationId();
-		
-		// verify access permission
-		if (!authorizationManager.canAccess(userInfo, evalId, ObjectType.EVALUATION, ACCESS_TYPE.UPDATE)) {
-			throw new UnauthorizedException("User ID: " + principalId +
-					" is not authorized to modify Submission ID: " + submissionId);
-		}
+		validateEvaluationAccess(userInfo, evalId, ACCESS_TYPE.UPDATE);
```
Comment (eric-wu) [inline services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/SubmissionManagerImpl.java]: Again DELETE?

Comment (bkng) [inline services/repository-managers/src/main/java/org/sagebionetworks/evaluation/manager/SubmissionManagerImpl.java]: whoops, missed that. should be fixed now

**Pull request merged**